### ISSUE TITLE
Informer stores improvements

### ISSUE THREAD

**Issue by deboer-tim** [containers/podman-desktop#5311]
### Is your enhancement related to a problem? Please describe

I've been testing with the ingress informer and draft PR for other informers. When they're connected they work excellent - changes are fast and responsive, feels way better than polling/watches. However, there are three things that need improvements:

- Because the informer stores don't use the normal stores/registry they don't show up on the Troubleshooting page. It would be nice if we could change them to naturally show up there when debugging, otherwise we need to add a new section.
- When I start with a running cluster things are fine, but if I start with (e.g.) the current context pointing to a Kind cluster that's stopped and then start it, the informer doesn't automatically sync. If I switch to another page and back it's usually fine though. We could use an event for internal/known clusters but not in general, so we likely need polling or some other mechanism to do an initial kick.
- If the cluster is stopped/disconnected it's really nice that the store isn't lost/reset, but we need some kind of disconnection indication in the UI.

### Describe the solution you'd like

See description.

**Comment by deboer-tim** [comment]
For the three issues listed in the description, I am only going to resolve the second here:
1. **Stores not showing up in troubleshooting** - IMHO we are likely to switch to the new/context informers which do show up, so this issue should be deferred and reevaluated later.
2. **Informers not connecting** - I investigated this today and it is not what I thought: if we attempt to start the informers before we can connect to the current context, getCurrentNamespace() is undefined and we skip creating the informer and never try again. I am working on a PR to change all of the createXInformer() functions to connect _where the context will be_ to avoid this problem, which is what the context informers do.
3. **Connection status** - This is now covered separately by issue #5401.

**Comment by deboer-tim** [comment]
I have a fix that works for the initial problem, but while building it I found other connection-related issues with the initial informers, and noticed several things that our newer informers handle (e.g. reconnection, backoff, jitter) that we'd also want to fix. Instead of doing this work twice, our focus has shifted to using the new informers everywhere and I'm closing this in favour of #6259.
